src/autonlp/autonlp.py

```diff
@@ -73,7 +73,7 @@ def _login_from_conf(self):
                         self.orgs = conf_json["orgs"]
                         self.token = conf_json["token"]
 
-    def create_project(self, name: str, task: str, language: str, max_models: int):
+    def create_project(self, name: str, task: str, language: str, max_models: int, hub_model: str = None):
```

Comment:
Might be time to add proper docstrings in the package!

Reply:
maybe we should add all docstrings in a single or. this is limited to cli and cli has hints.